OWML.ModHelper/ModInputHandler.cs

```diff
@@ -0,0 +1,447 @@
+﻿using System;
+using System.Reflection;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OWML.Common;
+using UnityEngine;
+
+namespace OWML.ModHelper
+{
+	public class ModInputHandler:IModInputHandler
+	{
+		private class InputInterceptor
+		{
+			public static void SAxisRemPre(SingleAxisCommand __instance)
+			{
+				ModInputHandler._self.UnregisterGamesBinding(__instance);
+			}
+
+			public static void DAxisRemPre(DoubleAxisCommand __instance)
+			{
+				ModInputHandler._self.UnregisterGamesBinding(__instance);
+			}
+
+			public static void SAxisUpdPost(SingleAxisCommand __instance)
+			{
+				KeyCode pos, neg;
+				ModInputHandler._self.RegisterGamesBinding(__instance);
+				FieldInfo val = typeof(SingleAxisCommand).GetField("_value", BindingFlags.NonPublic | BindingFlags.Instance);
+				float curval = (float)(val.GetValue(__instance));
+				int axisdir = 1;
```

Comment:
This whole file (not just this class) has lots of unnecessary abbreviations, and non-camelCase variables.

Reply:
hopefully found everything